ucan/datamodel/ucan/ucan.ipldsch

```diff
@@ -22,7 +22,7 @@ type Capability struct {
   with String
   can String
   # Any additional domain specific details and/or restrictions of the capability
-  nb Any
+  nb optional Any
```

Comment:
I'd also like either @hannahhoward or @alanshaw to sign off on this change as I don't fully grok the downstream effects of it.

Reply:
I think I tried this before and ended up just creating delegations with empty caveats (`ucan.NoCaveats`).

I think you need to depend on this in `go-libstoracha` and check the capabilities compile and test run.

Reply:
Confirmed: `go-libstoracha` passes with `github.com/storacha/go-ucanto@v0.4.1-0.20250618202138-28047da7acd7`.